recipes/rb-escape_utils/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
TODO: We should check if the unpacked gem (`gem unpack {{ name }}-{{ version }}.gem`) contains the license.

Reply:
The gem file actually contains the LICENSE:

```
gem unpack ~/Downloads/escape_utils-1.3.0.gem
Unpacked gem: '/Users/janjagusch/escape_utils-1.3.0'

ls escape_utils-1.3.0/LICENSE
escape_utils-1.3.0/LICENSE
```

So we should be able to make a slight adjustment in the recipe and then don't have to put the LICENSE manually into the feedstock.

Reply:
@svengiegerich, you can delete this file now.